tfutils/base.py

```diff
@@ -638,21 +676,25 @@ def train(sess,
             If loss exceeds this during training, HiLossError is thrown
     """
 
-    start_queues(sess, queues)
     step = global_step.eval(session=sess)
-    if step == 0:
-        log.info('Evaluating initial ...')
-        pass_targets = {k:v for k,v in train_targets.items() if k != 'optimizer'}
-        dbinterface.start_time_step = time.time()
-        train_results = sess.run(pass_targets)
-        dbinterface.save(train_results, {})
-        log.info('... done saving initial.')
 
-    if step < num_steps:
-        log.info('Training beginning ...')
-    else:
+    if num_steps is None:
+        num_steps = np.inf
```

Comment:
So what was the issue here with having `num_steps=np.inf` in the method call? Also, why not simply `num_steps=inf`?